.github/workflows/checkAndSubmitAddonMetadata.yml

```
@@ -278,16 +263,40 @@ jobs:
         jqExitCode=$?
         rm discussions.old.json
         exit $jqExitCode
+    - name: Create review comment
+      id: createComment
+      if: always()
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const fs = require('fs')
+          const addonVersion = "${{ steps.getAddonNameAndVersion.outputs.addonVersion }}"
+          const contents = fs.readFileSync('discussions.json')
+          const data = JSON.parse(contents)
+          const addonName = $ {{ steps.getAddonNameAndVersion.outputs.addonName }}
```

Fix syntax error raised in https://github.com/nvaccess/addon-datastore/pull/1757#issuecomment-1760236092

```suggestion
          const addonName = "${{ steps.getAddonNameAndVersion.outputs.addonName }}"
```